webpack.config.js

```diff
@@ -20,33 +20,8 @@ module.exports = {
     rules: [
       {
         test: /\.ts$/,
-        use: {
-          loader: 'babel-loader',
-          options: {
-            compact: true,
-            presets: [
-              ['@babel/preset-env', {modules: false}],
-              '@babel/preset-typescript',
-              ['@babel/preset-react', {runtime: 'automatic'}]
-            ],
-            plugins: [
-              'add-react-displayname',
-              [
-                'babel-plugin-styled-components',
-                {
-                  // Disable the dev-friendly classNames on styled-components
-                  displayName: !isProduction,
-                  // Minify the CSS
-                  minify: true,
-                  // Helps with dead code elimination
-                  // https://www.styled-components.com/docs/tooling#dead-code-elimination
-                  pure: true
-                }
-              ]
-            ]
-          }
-        },
-        exclude: [/node_modules/]
+        use: 'ts-loader',
+        exclude: /node_modules/
```

Comment:
This was what we used to have before the react 19 upgrade

Reply:
Why don't we also use `ts-loader` for the tsx files?

Reply:
I'm not sure, but this is how it was before and I didn't want to break anything again..

I tried it and it worked.. the only noticeable thing is it take a little longer to compile as ts-loader is slower than babel-loader